src/ApiService/ApiService/onefuzzlib/ReproOperations.cs

```diff
@@ -90,15 +90,15 @@ public ReproOperations(ILogTracer log, IOnefuzzContext context)
         var vm = await GetVm(repro, config);
         var vmOperations = _context.VmOperations;
         if (!await vmOperations.IsDeleted(vm)) {
-            _logTracer.Info($"vm stopping: {repro.VmId}");
+            _logTracer.Info($"vm stopping: {repro.VmId:Tag:VmId} {vm.Name:Tag:VmName}");
             var rr = await vmOperations.Delete(vm);
-            if (!rr) {
-                _logTracer.Error($"failed to delete repro {repro.VmId:Tag:VmId}");
+            if (rr) {
+                _logTracer.Info($"repro vm fully deleted {repro.VmId:Tag:VmId} {vm.Name:Tag:VmName}");
```

Comment:
Out of scope for this PR, but what do you think of enforcing some sort of pre+result logging in each of the `*Operations` functions?

That way we don't need to remember to wrap the call site in logs for starting/succeeding/failing an operation and we get the benefit of traceability everywhere else for free.

By that I mean...

**VmOperations.cs**:
```csharp
Delete(Vm vm) {
    _logTracer.Info($"vm stopping: {repro.VmId:Tag:VmId} {vm.Name:Tag:VmName}");
    ....
    // log if failed or succeded
    return result;
}
```

**ReproOperations.cs**
```csharp
Stopping(Repro repro) {
    // no need to log starting operation here
    await vmOperations.Delete(vm).IgnoreResult();
    // no need to log succeeded/failed, only use the result
}
```

Reply:
I'll start a discussion in the repo to get feedback from the team.

Reply:
good point